simulator/internal/internal.go

```diff
@@ -14,7 +14,7 @@ package internal
 // // Store NVDATA in memory, and we don't care about updates to failedTries.
 // #cgo CFLAGS: -DVTPM=NO -DSIMULATION=NO -DUSE_DA_USED=NO
 // // Flags from ../ms-tpm-20-ref/TPMCmd/configure.ac
-// #cgo CFLAGS: -std=gnu11 -Wall -Wformat-security -fstack-protector-all -fPIC
+// #cgo CFLAGS: -std=gnu11 -Wall -Wformat-security -fPIC
```

Comment:
jw: why won't this work on windows or macOS?

Reply:
This flag actually works on macOS and Linux. I changed the code to just only have the flag present on `!windows`.